packages/core/quri_parts/core/utils/bit.py

```diff
@@ -8,6 +8,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from typing import Union
+
+import numpy as np
+
+
+def bit_length(bits: Union[int, np.int8, np.int16, np.int32, np.int64]) -> int:
+    return int(bits).bit_length()
```

Comment:
Shall we add a docstring? (Though it is rather trivial)

Reply:
I have added the docstring for it.